protocol-models/src/main/resources/airbyte_protocol/v0/airbyte_protocol.yaml

```diff
@@ -485,6 +497,20 @@ definitions:
           type: array
           items:
             type: string
+  StreamMetadata:
```

Comment:
Shouldn't this be added into ConfiguredAirbyteStream?

Reply:
+1

Reply:
not to get too into bike shedding, but at the point that it is in `ConfiguredAirbyteStream` does it need its own struct at all? can't these all just be fields directly in `ConfiguredAirbyteStream`?